tools/export/uvision/__init__.py

```diff
@@ -247,7 +247,7 @@ def generate(self):
             sct_path, dirname(sct_name))
         if ctx['linker_script'] != sct_path:
             self.generated_files.append(ctx['linker_script'])
-        ctx['cputype'] = ctx['device'].core.rstrip("FD").replace("-NS", "")
+        ctx['cputype'] = ctx['device'].core.replace("-NS", "").rstrip("FDE")
```

Comment:
```suggestion
        ctx['cputype'] = ctx['device'].core.replace("-NS", "").rstrip("FD")
```

Maybe it should be without the E

Reply:
I dont understand  this change. How FD to FDE fixes it? why additional E?

Reply:
we got FD or F, no FDE? https://github.com/ARMmbed/mbed-os/blob/master/tools/export/uvision/__init__.py#L218